Source/DiscoveryV1/Models/NluEnrichmentCategories.swift

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright IBM Corp. 2018.
+ * (C) Copyright IBM Corp. 2018, 2019.
```

Comment:
Looks like my copyright linter is not quite working right.  This file ... and maybe some others ... shouldn't have a 2019 copyright because they haven't changed.